2EjercicioItems/Ammo.java

```diff
@@ -0,0 +1,52 @@
+import java.util.UUID;
+
+class Ammo implements Item{
+    UUID id=UUID.randomUUID(); // ???
```

Comment:
yeah this is good, you can leave it like this `UUID id=UUID.randomUUID();` or you can change it to `UUID id;`, either one is fine